From 27b3a6f00f121a0f8195e25c40faf97b57e2cec1 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca.weiss@fairphone.com>
Date: Mon, 28 Jul 2025 13:40:48 +0200
Subject: [PATCH 3/3] storage: Add modem_study into partition_table

The partition /boot/modem_study needs to be served on the Milos/SM7635
Fairphone (Gen. 6) smartphone.

Signed-off-by: Luca Weiss <luca.weiss@fairphone.com>
---
 storage.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/storage.c b/storage.c
index 140c6bf..001464d 100644
--- a/storage.c
+++ b/storage.c
@@ -41,6 +41,7 @@ static const struct partition partition_table[] = {
 	{ "/boot/modem_fs2", "modem_fs2", "modemst2" },
 	{ "/boot/modem_fsc", "modem_fsc", "fsc" },
 	{ "/boot/modem_fsg", "modem_fsg", "fsg" },
+	{ "/boot/modem_study", "modem_study", "study" },
 	{ "/boot/modem_tunning", "modem_tunning", "tunning" },
 	{ "/boot/modem_tng", "modem_tng", "tunning" },
 	{}
-- 
2.52.0

